app/models/user.rb

```diff
@@ -35,9 +35,15 @@ class User < ActiveRecord::Base
   include Facebookeable
 
   validates :username, uniqueness: true, allow_blank: true, allow_nil: true
+  validates_uniqueness_of :email, allow_blank: true, allow_nil: true
```

Comment:
It is better the general `validates` like the line 40: `validates :email, uniqueness: true, allow_blank: true, allow_nil: true`